dali/kernels/normalize/normalize_gpu.cu

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/kernels/normalize/normalize_gpu.h"
+#include <memory>
```

Comment:
Doesn't lint protest?

Reply:
No, it actually encourages this - there's a special rule for the header with a name same as source file (when only extension differs). It does protest in tests, because of _test suffix.